src/Dijkstra/AST/Prelude.agda

```diff
@@ -0,0 +1,50 @@
+module Dijkstra.AST.Prelude where
+
+open import Agda.Builtin.Unit
+     public
+open import Data.Bool                   using (Bool; true; false; not)
```

Comment:
I see a pattern of "only using" is on same line, otherwise, if there are other things (like renaming), on a separate line.

I think, in this file, do "using" on a separate indented line.

Reply:
DONE